x-pack/packages/observability/alert_details/src/hooks/use_alerts_history.test.tsx

```diff
@@ -59,6 +59,7 @@ describe('useAlertsHistory', () => {
     expect(result.current.isSuccess).toBeFalsy();
     expect(result.current.isLoading).toBeFalsy();
   });
+
   it('returns no data when API error', async () => {
```

Comment:
For these tests, despite the fact that the tests pass, I see the following error in the console, any idea how to solve it?
```
Warning: Can't perform a React state update on an unmounted component. 
This is a no-op, but it indicates a memory leak in your application. 
To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
```

![image](https://github.com/elastic/kibana/assets/12370520/08a91e64-2180-4239-bdb0-078d06f42320)